schemas/content/product.schema.json

```diff
@@ -60,111 +59,50 @@
           "description":
             "The internal unique ID of the product in the commerce backend system."
         },
-        "xdm:masterProductSKU": {
-          "title": "Master Product SKU",
-          "type": "string",
+        "xdm:prices": {
```

Comment:
How do we handle each variant having a different price?

Reply:
Since variants are product objects and use the same schema the can also have prices as well. If a variant has a same price like product but with different value than it can be set on variant level with same price type. 

For example a shirt with 5 different sizes defines the sales price (=type) on product level to USD 29 and variant with size S is USB 19. For this variant you define the sales price. All other variants inherit the sales price from the product.